_schedule/talks/2017-08-15-17-00-t1-django-vs-flask.md

```diff
@@ -23,7 +23,7 @@ presenters:
   website: https://www.davidbaumgold.com/
 room: 'Ballroom A'
 sitemap: true
-slides_url: ''
+slides_url: 'https://t.co/NiTcld5VGV'
```

Comment:
Should we switch this to the non-shortened URL? I know short links from some services expire after a bit. Here's the full link: https://archive.org/details/djangocon-us-2017-django-vs-flask